### ISSUE TITLE
Availability to rename torrent

### ISSUE THREAD

**Issue by Valicek1** [rndusr/stig#89]
It would be nice to be able to change torrent name like it is possible in transmission web interface. 
It uses json/rpc api, example:

```json
{
   "method":"torrent-rename-path",
   "arguments":{
      "ids":[341],
      "path":"Old Torrent Name",
      "name":"New Torrent Name"
   }
}
```

**Comment by Valicek1** [comment]
I know it's common to attach PR alongside with feauture request, but I am not python guy (:-1: for me)

**Comment by rndusr** [comment]
Yup, that's very high on my todo list.

Probably the next thing I'll work on after I'm done with polishing `search` and
`interactive`.

**Comment by rndusr** [comment]
No, users should request features or changes so the developers know where their
project is lacking. They can always ask for help, a full PR or simply reject the
request on a case-by-case basis.

Please don't hesitate to provide feedback. It's a form of contribution.

**Comment by Valicek1** [comment]
Thanks, built now from package stig-git (Archlinux AUR) - works like a charm!

**Comment by peaveyman** [comment]
I may have ran across a bug with this. I have torrents that are seeding to different trackers from the same folder. No problem there. When I try to rename a torrent to a name that already exists in the torrents it will not let me. Gives me this: torrent already exists in /torrents. I figure it's because the folder name is already there. Is there a way to override this?

**Comment by rndusr** [comment]
Yeah, I've run into this, too. The problem is that Transmission just overwrites
everything if it can or doesn't if it can't, but doesn't report back any result.

My workaround is to move the torrent somewhere else, rename it and move it back.

**Comment by peaveyman** [comment]
I just tried that, seems to work fine.
